### ISSUE TITLE
jQuery Security Vulnerability

### ISSUE THREAD

**Issue by ChemiKyle** [vanderbilt-redcap/custom-user-roles#1]
https://github.com/vanderbilt-redcap/custom-user-roles/blob/87b669575593325dabf9296946841b584d5ba291/manage_roles.php#L114

jQuery versions 3.4 and lower are [vulnerable to XSS attacks](https://security.snyk.io/package/npm/jquery/3.4.1), please update to 3.5.0 or later.

**Comment by mmcev106** [comment]
@moorejr5, are you able to knock this out?  If so, I'm happy to review it.

**Comment by moorejr5** [comment]
> @moorejr5, are you able to knock this out? If so, I'm happy to review it.

Yeah. There's a few of them so when I have them set up I'll get you tagged.
